addon/waiter-manager.ts

```diff
@@ -32,6 +32,16 @@ export function getWaiters(): IWaiter[] {
   return [...WAITERS.values()];
 }
 
+/**
+ * Puts waiters back into a default state where
+ * no waiter is pending.
+ *
+ * @public
+ */
+export function resetPendingWaiters(): void {
+  WAITERS.forEach(waiter => waiter.reset());
```

Comment:
This isn't a valid operation, there is no `reset` method on either the `ITestWaiter` or `IWaiter` interfaces. The type for `WAITERS` is not setup correctly we need to change from:

```js
const WAITERS = new Map();
```

To:

```js
const WAITERS = new Map<string, IWaiter>();
```

Reply:
This was simply an omission on my part. I'll add.

Reply:
I can update in a separate PR.